### PR TITLE
Add new bind role 2.6

### PR DESCRIPTION
**Pull request by oafischer**
https://github.com/stolostron/backlog/issues/25778

### CODE REVIEW

File: access_control/rbac.adoc [hunk 1]
```diff
@@ -53,6 +53,9 @@ Some product resources are cluster-wide and some are namespace-scoped. You must
 
 | *admin, edit, view*
 | Admin, edit, and view are {ocp-short} default roles. A user with a namespace-scoped binding to these roles has access to `open-cluster-management` resources in a specific namespace, while cluster-wide binding to the same roles gives access to all of the `open-cluster-management` resources cluster-wide.
+
+| *open-cluster-management:managedclusterset:bind:<managed_clusterset_name>*
+| A user with the `open-cluster-management:managedclusterset:view:<managed_clusterset_name>` role has view access to the managed cluster resource called `<managed_clusterset_name>`. The user can bind `<managed_clusterset_name>` to a namespace. The user also has view access to `managedcluster.cluster.open-cluster-management.io`, `clusterclaim.hive.openshift.io`, `clusterdeployment.hive.openshift.io`, and `clusterpool.hive.openshift.io` resources, which have the following managed cluster set labels: `cluster.open-cluster-management.io`, `clusterset=<managed_clusterset_name>`. See link:../multicluster_engine/cluster_lifecycle/managedclustersets_intro.adoc#managedclustersets-intro[Creating and managing ManagedClusterSets] to learn how to manage the resource.
```
Comment (ldpliu) [inline access_control/rbac.adoc]: 1. `open-cluster-management:managedclusterset:view:<managed_clusterset_name>` should changed to `open-cluster-management:managedclusterset:bind:<managed_clusterset_name>`
2. "which have the following managed cluster set labels: `cluster.open-cluster-management.io`, `clusterset=<managed_clusterset_name>`."  should changed to "which have the following managed cluster set labels: `cluster.open-cluster-management.io/clusterset=<managed_clusterset_name>`."  (same change should be applied to `| *open-cluster-management:managedclusterset:admin:<managed_clusterset_name>*` and  | *open-cluster-management:managedclusterset:view:<managed_clusterset_name>* section)

**Comment by oafischer** [comment]
Implemented the feedback, waiting for writing review as well.

**Comment by ldpliu** [comment]
> Implemented the feedback, waiting for writing review as well.

The second change should be also applied to `open-cluster-management:managedclusterset:admin:<managed_clusterset_name>` and `open-cluster-management:managedclusterset:view:<managed_clusterset_name>` section
https://github.com/stolostron/rhacm-docs/pull/4133#discussion_r980645511

**Comment by oafischer** [comment]
Should be correct now @ldpliu, thanks!

File: access_control/rbac.adoc [hunk 2]
```diff
@@ -43,7 +43,7 @@ Some product resources are cluster-wide and some are namespace-scoped. You must
 | A user with cluster binding to the `open-cluster-management:view:<managed_cluster_name>` role has view access to the `ManagedCluster` resource named,  `<managed_cluster_name>`.
 
 | *open-cluster-management:managedclusterset:admin:<managed_clusterset_name>*
```
Comment (swopebe) [inline access_control/rbac.adoc]: I am not sure why these are bold...I know it's not your decision. They should be in code block...I saw these a lot this release, just did not have time to change them. 

should all be: `open-cluster-management:managedclusterset:admin:<managed_clusterset_name>`

I made them all the same...either code block or nothing applies, but bold is very selective and can easily get misused.

**Review by swopebe — APPROVED** [review]

**Review by ldpliu — APPROVED** [review]

**Pull request merged**